frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -224,12 +214,15 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
             </FormSection>
           </StackItem>
         </Stack>
+        <DashboardModalFooter
```

Comment:
PF offers a prop for Modal named `footer`, can you pass the footer component into it instead of render it in the children? I have an example here https://github.com/opendatahub-io/odh-dashboard/blob/f/mserving-metrics/frontend/src/pages/modelServing/screens/metrics/biasConfigurationModal/ManageBiasConfigurationModal.tsx#L71-L82

Reply:
And also, in this way, I don't think you need the `Stack` layout in the Modal children anymore to stack the form groups. Just simply put all the sections together, and remove all the `Stack` and `StackItem` components here

Reply:
@pnaik1